### DISCUSSION TITLE
Under the ARM64 architecture

### DISCUSSION THREAD

**Discussion by llody55** [rabbitmq/rabbitmq-server#6518] [Other]
Under the ARM64 architecture, rabbitmq cannot join the cluster correctly, and the page nodes switch automatically, but all the nodes in the cluster are not displayed. Version 3.7.26+ involved

**Comment by michaelklishin** [comment]
We cannot suggest much with this problem statement. Consider sharing logs.

RabbitMQ does not have any architecture-specific code. There is a known bug in Erlang 25 that makes it segfault on ARM 64 but 3.7 won't run on 25 anyway.

FYI, RabbitMQ 3.7 has reached [end of life over two years ago](https://www.rabbitmq.com/versions.html).
